ImperatorToCk2/Characters.java

```diff
@@ -0,0 +1,208 @@
+import java.util.Scanner;
+import java.io.IOException;
+import java.io.FileInputStream;
+import java.io.PrintWriter;
+import java.io.FileOutputStream;
+import java.util.Random;
+/**
+ * Everything dealing with characters
+ *
+ * @author (your name)
+ * @version (a version number or a date)
+ */
+public class Characters
+{
+    private int x;
+
+    public static String[] importChar (String name, String charID) throws IOException
+    {
+
+        String tab = "	";
+        char quote = '"';
+
+        FileInputStream fileIn= new FileInputStream(name);
+        Scanner scnr= new Scanner(fileIn);
+
+        int flag = 0;
+
+        String keyWord = charID+"={";
+
+        int aqq = 0;
+
+        boolean endOrNot = true;
+        String vmm = scnr.nextLine();
```

Comment:
This variable isn't really used. Drop it and assign where you really care. Or if you're dropping a line, assign it to a variable called unused.